src/Sarif.Multitool.Library/Rules/GH1005.LocationsMustBeRelativeUrisOrFilePaths.cs

```diff
@@ -4,8 +4,9 @@
 using System;
 using System.Collections.Generic;
 using System.Linq;
-
+using System.Linq.Expressions;
 using Microsoft.Json.Pointer;
```

Comment:
we have a style configuration to enforce the using by groups. If you can check the dotnet-format check you will find some issues. Can you resolve those?